chat-commands.js

```diff
@@ -895,7 +895,7 @@ exports.commands = {
 		if (!target) return;
 		if (!/</.test(target)) {
 			// not HTML, do some simple URL linking
-			let re = /(https?:\/\/(([-\w\.]+)+(:\d+)?(\/([\w/_\.]*(\?\S+)?)?)?))/g;
+			let re = /(https?:\/\/(([-\w.]+)+(:\d+)?(\/([\w/_.]*(\?\S+)?)?)?))/g;
```

Comment:
Here's another block where I'd prefer to end with `-]` rather than starting with `[`